features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/modules/manager/manager.jag

```diff
@@ -160,5 +160,9 @@ var getActiveTenantDomains=function(){
 
 };
 
+var getUsageClient = function(){
```

Comment:
fixed in commit: https://github.com/wso2/carbon-apimgt/commit/9619da2271b49f37ef93456b6f8a8ac235f685ea